sources/postgres/infoschema.go

```diff
@@ -103,7 +103,7 @@ func (isi InfoSchemaImpl) GetRowsFromTable(conv *internal.Conv, tableId string)
 	// PostgreSQL schema and name can be arbitrary strings.
 	// Ideally we would pass schema/name as a query parameter,
 	// but PostgreSQL doesn't support this. So we quote it instead.
-	isSchemaNamePrefixed := strings.HasPrefix(conv.SrcSchema[tableId].Name, conv.SrcSchema[tableId].Schema)
+	isSchemaNamePrefixed := strings.HasPrefix(conv.SrcSchema[tableId].Name+".", conv.SrcSchema[tableId].Schema)
```

Comment:
Isn't the "." appended opposite ? Shouldn't it be appended to Schema name for verification?

Reply:
Oops. I've updated it now.